### ISSUE TITLE
Fix `CssSyntaxError` for left bracket without leading space

### ISSUE THREAD

**Issue by JounQin** [stylelint/stylelint#5505]
> Clearly describe the bug

As title

> Which rule, if any, is the bug related to?

N/A

> What code is needed to reproduce the bug?

```less
@import 'variables';

.strong-password {
  &__tips {
    max-width: none;
    font-size: @font-size-xs;

    // error when no leading space here, pass when adding a space
    .@{ant-prefix}-tooltip{
      &-arrow {
        display: none;
      }

      &-inner {
        padding: 12px;
        background-color: #fff;
        color: @text-color-secondary;
      }
    }

    &__title {
      color: @text-color;
    }
  }
}
```

> What stylelint configuration is needed to reproduce the bug?

e.g.

```json
{
  "extends": ["@1stg/stylelint-config", "@1stg/stylelint-config/modules"]
}
```

> Which version of stylelint are you using?

`13.13.1`

> How are you running stylelint: CLI, PostCSS plugin, Node.js API?

CLI with `stylelint . --cache`

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

Yes, it's related to Less variable in property

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

The following warnings were flagged:

```shell
src/components/StrongPassword/index.less
 24:1  ✖  Unexpected }   CssSyntaxError
```

**Comment by jeddy3** [comment]
@JounQin Thanks for the report and for using the template.

It's likely a problem with the parser stylelint uses to support Less. Please consider [contributing to the parser](https://stylelint.io/developer-guide/syntaxes#working-on-syntaxes) if you have time.
